systers_portal/templates/meetup/base.html

```diff
@@ -1,14 +1,14 @@
 {% extends "base.html" %}
 
 {% block title %}
-  {{ object.name }}
+  {{ slug|capfirst }} Systers
```

Comment:
This is in base template which can be served to DetailView, ListView, or any other edit view where we don't have `object` instance its only in DetailView we have it. So for those templates that will get data from ListView or any edit view and inherit this base template will not display Systers name.